src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -3757,18 +3757,27 @@ namespace HeatBalanceManager {
         }
 
     Label10:;
-        for (LineNum = 2; LineNum <= 5; ++LineNum) {
+        NextLine = W5DataFile.readLine();
+        if (NextLine.eof) {
+            goto Label1000;
+        }
+        ++FileLineCount;
+        if (!has_prefixi(NextLine.data, "WINDOW NAME")) {
```

Comment:
Although I'm not a big fan of adding another use of the Objexx utilities, this is totally fine.